lib/myxql.ex

```diff
@@ -100,6 +101,8 @@ defmodule MyXQL do
       will disconnect the connection. See "Disconnecting on Errors" section below for more
       information.
 
+    * `:enable_cleartext_plugin` - Set to `true` to send password as cleartext (default: `false`)
```

Comment:
I think matching mysql CLI is a great starting point but curious if any other connectors have something like `auth_plugin: mysql_native_password  | caching_sha2_password | ...`. I think that'd be preferred as we'd reuse the same option when we support more methods.